added info on --non-interactive-include-reboot-patches (#759)
SUSE/doc-caasp: adoc/admin-updates.adoc

=== Disabling Automatic Updates

Nodes added to a cluster have the service `skuba-update.timer`, which is responsible for running automatic updates, activated by default.

This service calls the `skuba-update` utility and it can be configured with the `/etc/sysconfig/skuba-update` file.
[NOTE]
===

> `skuba-update` uses the flags `--non-interactive --non-interactive-include-reboot-patches`. `--non-interactive-include-reboot-patches`, when used in conjunction with `--non-interactive`, will cause patches with the `rebootSuggested-flag` not to be skipped. The reboot will be managed by `kured`.

Reviewer: ```suggestion
`skuba-update` uses the flags `--non-interactive` and `--non-interactive-include-reboot-patches`. The `--non-interactive` flag causes zypper to use default answers to questions rather than prompting a user.  In non-interactive mode, the `--non-interactive-include-reboot-patches` flag causes patches with the `rebootSuggested-flag` to not be skipped. Zypper does not perform the reboot directly. Instead, `kured` will be used to safely schedule reboots as needed.
```

Minor rearrangement to avoid starting a sentence with a hyphen, and some similar grammatical tweaks.

Reviewer 2: Looks ok, thanks.

> `skuba-update` uses the flags `--non-interactive --non-interactive-include-reboot-patches` when running. `--non-interactive-include-reboot-patches` when used in conjunction with `--non-interactive` this will treat patches with the `rebootSuggested-flag` as non-interactive as the reboot will be managed by `kured`.

Reviewer 3: ```suggestion
`skuba-update` uses the flags `--non-interactive --non-interactive-include-reboot-patches`. `--non-interactive-include-reboot-patches`, when used in conjunction with `--non-interactive`, will cause patches with the `rebootSuggested-flag` not to be skipped. The reboot will be managed by `kured`.
```

Author: :bow:

> `skuba-update` uses the flags `--non-interactive --non-interactive-include-reboot-patches` when running. `--non-interactive-include-reboot-patches` when used in conjunction with `--non-interactive` this will treat patches with the `rebootSuggested-flag` as non-interactive as the reboot will be managed by `kured`.

Reviewer 4: You could drop the word "this" in "this will treat patches" as the subject was just stated and it reads fine without the "this".
I might rephrase "as non-interactive as the reboot will be managed by `kured`" to "as non-interactive allowing the reboot to be managed by `kured`".  Having "as" used twice in the sentence for two slightly different meanings just read a little funny in my mind.  But it is ok either way.

Author: please read it now and let me know what you think. @cmurphy 's commit corrected the problem I think.
